script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_3_1.txt

```diff
@@ -1,24 +1,24 @@
 <Page1>
 "...Well, you're not wrong. 
 I'd really prefer a pair though. Two women or a man and a woman."
-"It's fine. Welcome to the club, Otaku-kun!
-We've been troubled by our lack of challengers lately! If we hadn't met our quota, we would have been killed too!"
+"It's fine though. Welcome to the club, Otaku-kun!
+We've been troubled by the lack of challengers lately! If we don't meet our quota, we'll be killed too!"
 <Page2>
 Letting out a cackle, the man called Haru circles around to my back.
 He blocks my path of retreat.
 <Page3>
 "Well, I guess it's fine to have a single guy once in a while.
 I'll take you to the part-time venue."
-The man named Haru pushes the button in the elevator. 
+The man named Haru pushes a button in the elevator. 
 <Page4>
 A mechanical sound echoes in the darkness. 
 This seems to be the only place that has electricity.
 <Page5>
 "Get in." 
 "......" 
-...I don't know if I should go into the elevator like this.
+...I don't know if I should go into the elevator.
 Once I enter, I'll be locked inside.
-If the two men were to seize me, I would not be able to resist with my physical strength.
+If the two men were to seize me, my physical strength wouldn't be enough to resist.
```

Comment:
original ver. of this line is better, but neither are that great.
> If the two men were to seize me, I wouldn't be able to resist with my current strength.

I think it's "physical strength" that's awkward in the original version.

Reply:
If both men were to seize me, I wouldn't be able to put up much of a fight.